### DISCUSSION TITLE
开启播放后似乎会锁定帧率

### DISCUSSION THREAD

**Discussion by BPTPW** [amll-dev/applemusic-like-lyrics#219] [Q&A] [answered]
网易云3.0.0版本
显卡：RTX 3060
CPU：RYZEN R5 5600

播放状态下歌词界面只有30帧
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/62642316/961b0af4-b44a-4dd5-816f-6d8cc667c263)
暂停后：
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/62642316/3d372774-50c1-43fa-afac-cc1a795e23a7)
同时此问题会影响整个网易云的帧率
播放时：
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/62642316/7208db67-8d63-4212-9f9e-3350c715ac6c)
暂停时：
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/62642316/cf434668-ef5b-406a-941e-9cb5ecfc42e8)

**Comment by Steve-xmh** [comment]
看一下程序占用

**Reply by BPTPW** [reply, marked as answer]
解决了，是在显卡的设置里调整了后台最大帧率的原因
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/62642316/2583689d-f3bc-410e-bb37-316d8df4ef0b)
不知道为什么应用在前台的时候也会受到这个设置的影响
